internal/container/dockerfile.go

```diff
@@ -109,7 +143,11 @@ func findDockerfilesAt(dir string) []string {
 		return nil
 	})
 
-	fmt.Println("Found Dockerfiles: " + strings.Join(foundFiles, ", \n\t"))
+	if len(foundFiles) >= 1 {
```

Comment:
not sure if we need that statement, nothing happens within it

Reply:
now it makes sense again